docs/jacob-bansal.md

```diff
@@ -0,0 +1 @@
+ Add the information from the Issue you created earlier to the file.
```

Comment:
add a line comment to the Pull Request mentioning that you should add your favorite color